### PR TITLE
Typo correction in comment

### PR DESCRIPTION
**Pull request by frankhereford**
This PR intends to correct a typo in the docstring for the `get` method in `app.py`.

### CODE REVIEW

File: knackpy/app.py [hunk 1]
```diff
@@ -205,7 +205,7 @@ def get(
                     API. Defaults to False.
                 record_limit (int): the maximum number of records to retrieve. If
                     `None`, will return all records.
-                filters (dict or list, optional): A dict or of Knack API filiters.
+                filters (dict or list, optional): A dict or of Knack API filters.
```
Comment (johnclary) [inline knackpy/app.py]: I actually have a second typo here (lol 🙈): the word "list" is missing.

> filters (dict or list, optional): A dict or _list_ of Knack API filters.

**Review by johnclary — DISMISSED** [review]
@frankhereford thanks for flagging this. we might as well rebuild the documentation website while we're at it (to keep the API reference in sync).

it's a bit cumbersome, but will be a good intro to the tooling for you. [these docs](https://cityofaustin.github.io/knackpy/docs/developer-guide/#documentation) cover that process.

unfortunately i do not have CI setup for the docs HTML. you're welcome to take a crack at that if you're feeling ambitious.

File: knackpy/app.py (hunk 1 above)
Comment (frankhereford) [reply knackpy/app.py]: My reliance on my spell checker in VS Code has never been more apparent! Without a squiggly line, I didn't even see that!

**Comment by frankhereford** [comment]
> @frankhereford thanks for flagging this. we might as well rebuild the documentation website while we're at it (to keep the API reference in sync).

> it's a bit cumbersome, but will be a good intro to the tooling for you. these docs cover that process.

> unfortunately i do not have CI setup for the docs HTML. you're welcome to take a crack at that if you're feeling ambitious.

I would love to take a shot at it! I will dig into it here in the next day or two, and I am looking forward to learning more about how that process works. Thanks for the suggestion to give it a whirl, and I'll let you know how it goes.

**Comment by johnclary** [comment]
@frankhereford i rebuilt the docs and am bringing em in! The PyPI will bomb because this will trigger a duplicate publication of the same `knackpy-dev` package version, which is fine because this is purely Github pages HTML*. [Related discussion](https://github.com/cityofaustin/knackpy/pull/110#issuecomment-1099868620).

Edit: right, this is a docstring edit. because we're not bumping version, it will be absent from `knackpy-dev`, but will be included in the next prod release.

**Pull request merged**